Workflow.Rmd

```diff
@@ -368,85 +370,351 @@ state_accounts_index %>%
+
+Often a generic random variable is denoted $X$,
+which should not be confused with the "knowns" in our model taxonomy above. A generic realization of a random variable
+by convention is in the lower case, so $x$ would be a realization of $X$. For example, $X$ might be "Sarah Smith's 
+potential income in 2018", while $x$ might be "the amount that Sarah Smith actually earned in 2018". Note here
+that the random variable here "Sarah's potential income" is entirely hypothetical---random variables are a 
+convenient way of representing and _idea_, and are not necessarily observed data. We'll get to the relationship between
+random variables and our data later in this section. 
+
+**All random variables have probability distributions**
+
+The point of using Bayesian methods is to understand the _joint probability distribution_ of a collection of random
+variables. Let's set aside the _joint_ part of this and briefly define a probability distribution. 
+A probability distribution, also known as a _probability density_ (for continuous random variables)
+or a _probability mass_ (for discrete random variables) is a function that describes the relative probability of observing
+various values of a random variable. Formally, a probability distribution is a function $p()$ that assigns 
+some non-negative value to each possible realization of a random variable, and sums (integrates) to 1 over all
+possible values of the random variable. That is, a probability distribution says that _some outcome_ must happen, 
+and describes the relative probabilites of each. If $p()$ assigns a value of 0 to some value of a random 
+variable (formally, $p(X=x) = 0$ for some $x$), we do not think it's possible to for $X$ to take on the 
+value $x$. If $p(X=x_{1}) > p(X= x_{2})$, it is more probable to observe a value of $X$ with the 
+value $x_{1}$ than $x_{2}$. 
+
+The notation for probability distributions $p()$ is _overloaded_, which is to say that if we have two 
+random variables $X$ and $Y$, then the function $p(X)$ is different to the function $p(Y)$. And neither
+of these function need to have a neat analytical form. When we can represent $p()$ with a mathematical
+expression with a fixed number of parameters, then we say that $p()$ is a *parametric* distribution 
+function. If we cannot express $p(X)$ analytically, then $p(X)$ is *nonparametric*. A great deal of 
+statistical modeling involves finding ways of approximating nonparametric distributions with parametric ones, 
+and finding ways of extracting information from nonparametric distributions efficiently. 
+
+If we know the distribution of a random variable with certainty, then we can also make probabilistic 
+statements about the random variable falling within a set of values. Yet we typically do not know the 
+distribution of the random variable, and have to infer what it might be. 
+
+
+
 From this distribution, we can make probabilistic statements, like "The probability of a randomly chosen male
```

Comment:
This is clearly in need of replacing. Above here is where I plan on putting something about the empirical distribution of incomes in the loans data.